src/panel/codicon.css

```diff
@@ -0,0 +1,426 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
```

Comment:
Is this also a candidate to retrieve using git in the build pipeline?

Reply:
Conceptually possible. However I don't have line-of-sight. Ideally, they would need to be fetched before the bundling process (to make it into the bundle) and I don't know the hook for that. Would need to research.

Alternatively, can drop it in the out folder, but it results in some extra loose files.

Reply:
Conceptually possible. However I don't have line-of-sight. Ideally, they would need to be fetched before the bundling process (to make it into the bundle) and I don't know the hook for that. Would need to research.

Alternatively, can drop it in the out folder, but it results in some extra loose files.

---
In reply to: [430586831](https://github.com/microsoft/sarif-vscode-extension/pull/279#discussion_r430586831) [](ancestors = 430586831)

Reply:
Good news: They just released these icons on NPM. Bad news: There is a blocking bug. I've filed an issue with them to track: [https://github.com/microsoft/vscode-codicons/issues/15](https://github.com/microsoft/vscode-codicons/issues/15). When they fix it, I'll complete this item.

---
In reply to: [436289721](https://github.com/microsoft/sarif-vscode-extension/pull/279#discussion_r436289721) [](ancestors = 436289721,430586831)

Reply:
They fixed the package. We are no longer checking these files in. Resolved.

---
In reply to: [436327157](https://github.com/microsoft/sarif-vscode-extension/pull/279#discussion_r436327157) [](ancestors = 436327157,436289721,430586831)